tests/events/package.json

```diff
@@ -15,5 +15,8 @@
   },
   "scripts": {
     "test": "anchor test"
+  },
+  "dependencies": {
+    "@coral-xyz/anchor": "^0.27.0"
```

Comment:
We'd like to run the tests against the latest version in repository. If you check the `yarn.lock` file, you'd see that this change makes it so that we test against the last released version on NPM instead of the latest version in the repository. Please remove this and update the lock file.

Reply:
Need to build ts packages first and link them to be used in tests, see https://github.com/coral-xyz/anchor/blob/master/.github/actions/setup-ts/action.yaml

We should definitely add a script for this though https://github.com/coral-xyz/anchor/issues/2471

Reply:
Thanks for this! This worked